Source/src/utils/RandomUtil.h

```diff
@@ -26,18 +26,21 @@ namespace Hachiko
         HACHIKO_API static float RandomBetween(const float2& values);
     };
 
-    class HACHIKO_API PerlinNoise1D
+    class HACHIKO_API PerlinNoise
     {
     public:
-        PerlinNoise1D();
-        PerlinNoise1D(unsigned int seed);
+        PerlinNoise();
+        PerlinNoise(unsigned int seed);
 
         float Random(float previous);
         float RandomRange(float previous, float min, float max);
         
+        [[deprecated]] float* GeneratePerlinNoise2d(int x, int y, int samples = 1); // not working right
+
```

Comment:
Let's just delete this if this is not working right, as you are using simplex already.

Reply:
Oka, but I would like to fix it :(